tests/cephfs/cephfs_perf/cephfs_io_read_write_from_multiple_clients.py

```diff
@@ -0,0 +1,160 @@
+                    client2,
+                    client_info["mounting_dir"],
+                    "g",
+                    "read",
+                )
+                p.spawn(
+                    fs_util.stress_io,
+                    client3,
+                    client_info["mounting_dir"],
+                    "",
+                    0,
+                    2,
+                    iotype="smallfile",
+                )
+                p.spawn(
+                    fs_util.stress_io,
+                    client2,
+                    client_info["mounting_dir"],
+                    "",
+                    0,
+                    2,
+                    iotype="smallfile",
+                )
+                p.spawn(fs_util.read_write_IO, client3, client_info["mounting_dir"])
+                for op in p:
+                    (return_counts, rc) = op
+            c1 = ceph_df(ceph_cluster)
+
+        check_health(ceph_cluster)
+        log.info("Test completed for CEPH-%s" % tc)
+        print("Results:")
+        result = fs_util.rc_verify(tc, return_counts)
+        print(result)
+        print("Script execution time:------")
+        stop = timeit.default_timer()
+        total_time = stop - start
+        (mins, secs) = divmod(total_time, 60)
+        (hours, mins) = divmod(mins, 60)
+        print("Hours:%d Minutes:%d Seconds:%f" % (hours, mins, secs))
+        return 0
+
+    except CommandFailed as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        log.info("Cleaning up!-----")
+        if client3[0].pkg_type != "deb":
+            rc = fs_util.client_clean_up(
+                client_info["fuse_clients"],
+                client_info["kernel_clients"],
+                client_info["mounting_dir"],
+                "umount",
+            )
+        else:
+            rc = fs_util.client_clean_up(
+                client_info["fuse_clients"], "", client_info["mounting_dir"], "umount"
+            )
+        if rc == 0:
+            log.info("Cleaning up successfull")
+
+    except Exception as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        return 1
+
+
+def ceph_df(ceph_cluster):
+    for mnode in ceph_cluster.get_nodes(role="client"):
+        if mnode.role == "client":
+            m = mnode.role
+            log.info("%s" % m)
+            out, rc = mnode.exec_command(cmd="sudo ceph df ")
+            l = []
+            for s in out.split():
+                if re.findall(r"-?\d+\.?\d*", s):
+                    l.append(s)
+            print(l[3])
+            cluster_filled_perc = float(l[3])
+            if cluster_filled_perc > 30:
+                return 0
+            return 1
+
+
+def check_health(ceph_cluster):
+    for mnode in ceph_cluster.get_nodes(role="client"):
+        if mnode.role == "client":
+            m = mnode.role
+            log.info("%s" % m)
+            out, rc = mnode.exec_command(cmd="sudo ceph -s -f json ")
+            out2 = out.split()
```

Comment:
you can remove line no: 154 and if loop can be
if out['health']['status'] == "HEALTH_OK":